$NetBSD: patch-ad,v 1.1 2007/12/01 15:54:28 rillig Exp $

This code is unused and contains references to undefined classes.

--- RasTbl.h.orig	2004-10-18 12:39:19.000000000 +0000
+++ RasTbl.h	2007-12-01 14:59:50.000000000 +0000
@@ -377,19 +377,6 @@ public:
 	endptr FindByAliases(const H225_ArrayOf_AliasAddress & alias) const;
 	endptr FindEndpoint(const H225_ArrayOf_AliasAddress & alias, bool RoundRobin, bool SearchOuterZone = true);
 
-	template<class MsgType> endptr getMsgDestination(const MsgType & msg, unsigned int & reason,
-	                                                 bool SearchOuterZone = true)
-	{
-	  endptr ep;
-	  bool ok = getGkDestAnalysisList().getMsgDestination(msg, EndpointList, listLock,
-	                                                      ep, reason);
-	  if (!ok && SearchOuterZone) {
-            ok = getGkDestAnalysisList().getMsgDestination(msg, OuterZoneList, listLock,
-	                                                   ep, reason);
-	  }
-	  return (ok) ? ep : endptr(0);
-	}
-
 	void ClearTable();
 	void CheckEndpoints();
 
